calcom/cal.com
# fix: Enable Salesforce & CRMs for teams & orgs [CAL-2116]

## What does this PR do?



This PR adds state that contains a teamId in order to install CRM apps to teams.

Fixes # (issue)



## Type of change



- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?



- Create a team
- Install Salesforce
- Create a booking
- The new attendee should be created in Salesforce

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
